<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListinoDlg</class>
 <widget class="QDialog" name="ListinoDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>216</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fornitore:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fornitoreCb"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Ricerca:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
      <item>
       <widget class="QRadioButton" name="printAllRb">
        <property name="text">
         <string>Stampa listino completo</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="printFromDateRb">
        <property name="text">
         <string>Stampa articoli con data corrente</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
        <item>
         <widget class="QRadioButton" name="printFromFatturaRb">
          <property name="text">
           <string>Stampa articolo con fattura:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Chiudi</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="configButton">
       <property name="text">
        <string>Stampante</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Salva</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>saveButton</sender>
   <signal>clicked()</signal>
   <receiver>ListinoDlg</receiver>
   <slot>draw()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>394</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>404</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>configButton</sender>
   <signal>clicked()</signal>
   <receiver>ListinoDlg</receiver>
   <slot>config()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>257</y>
    </hint>
    <hint type="destinationlabel">
     <x>322</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>ListinoDlg</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>printFromFatturaRb</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>136</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>draw()</slot>
  <slot>config()</slot>
 </slots>
</ui>
